### ISSUE TITLE
Interfacing BeagleBone Black with AD8232 Heart Rate Sensor

### ISSUE THREAD

**Issue by BibinBabu94** [jassjagga/Capstone-Project-summer_2020#7]

**Comment by mikeshams** [comment]
What issues did you or others might face with connection of the heart rate sensor to a BBB in terms of Hardware and Software?

**Comment by mikeshams** [comment]
Please remove the issue regarding very changeable numbers for the heart rate asap!

**Comment by BibinBabu94** [comment]
1. The analog out of BeagleBone Black is 1.8V and the output from AD8232 is 2 to 2.5 volts so I had to use a 1K ohms resistor in series in order to limit the voltage.
2. At first, I was not to read the analog data from my code then I figured out that for the connection to be complete I should connect the analog ground and the digital ground as well.
3. Another main issue I faced was with noise so I used a Dotted PCB in order to limit the noise and then set a threshold value in order to limit certain spike.
4. The final issue was with changing values so I tried increasing the delay time and then got almost same values but the ECG is varying for every time period
